test/e2e/resize_nodes.go

```diff
@@ -400,6 +405,21 @@ var _ = Describe("Nodes", func() {
 			waitForNodeToBe(c, node.Name, true, 2*time.Minute)
 
 			By("verify wheter new pods can be created on the re-attached node")
+
+			/*
+			 * I'm assuming here that the scheduler is 'monotonic': we scheduled
+			 * as many pods as we had nodes, and the node X was chosed to host
+			 * a pod (at least one). Then we failed node X and after that all
+			 * other nodes have more (or the same number of) pods as at the beginning
+			 * of the test. And the new node has no pods. I'm assuming that
+			 * the new pod, created just below, will be scheduled on node X.
+			 *
+			 * I think this assumption is true for schedulers that take into account
+			 * pod names.
```

Comment:
Please use the feature that we have designed for this. https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/types.go#L897